core/Symbols.cc

```diff
@@ -905,95 +901,40 @@ ClassOrModule::findMemberFuzzyMatchConstant(const GlobalState &gs, NameRef name,
 
     bool onlySuggestPackageSpecs = ref(gs).isPackageSpecSymbol(gs);
     Levenstein levenstein;
+    vector<ClassOrModuleRef> candidateScopes;
+    vector<ClassOrModule::FuzzySearchResult> scopeBest;
+
+    // Ensure that we start the search from the attached class -- we're interested in constants, which aren't present on
+    // the singleton.
+    ClassOrModuleRef base = this->isSingletonClass(gs) ? this->attachedClass(gs) : this->ref(gs);
```

Comment:
Slight preference to just use `attachedClass(gs).exists()` instead of `isSingletonClass`, not blocking.

Reply:
Yeah, I like that more. Thanks!